packages/api-explorer/src/routes/AppRouter.tsx

```diff
@@ -47,18 +49,33 @@ export const AppRouter: FC<AppRouterProps> = ({ specKey, api }) => {
           <OAuthScene />
         </Route>
       )}
-      <Route path="/:specKey/(methods|types)?" exact>
-        <HomeScene api={api} />
-      </Route>
-      <Route path="/:specKey/methods/:methodTag" exact>
-        <TagScene api={api} />
-      </Route>
-      <Route path="/:specKey/methods/:methodTag/:methodName">
-        <MethodScene api={api} />
-      </Route>
-      <Route path="/:specKey/types/:typeName">
-        <TypeScene api={api} />
-      </Route>
+      <Route
```

Comment:
Alternatively, create specific validation methods for home, method, type and tag. I also think the method should return a boolean rather than render the component. Something like
```
      <Route
        path="/:specKey/(methods|types)?"
        render={({ match }) => {
          return validHomeRoute(match, api)  ? <HomeScene api={api} /> : <NotFound/>
        }
        exact
      />